EzPC/EzPC/tc.ml

```diff
@@ -157,7 +167,16 @@ let check_non_void_ret_typ (f:string) (t:ret_typ) (r:range) :eresult =
   match t with
   | Typ t -> Well_typed t
   | Void _ -> Type_error ("Function " ^ f ^ " has a void return type", r)
-       
+
+let join_types_cmp (t1:typ) (t2:typ) :typ option =
```

Comment:
Is this used anywhere?

Reply:
Missed to remove it earlier, removed now.